extension/services/searchServices.toml

```diff
@@ -37,6 +37,7 @@ bangSearch = "az"
 
 [wikipedia]
 bangSearch = "wikipedia"
+baseUrl = "https://en.wikipedia.org/wiki/"
```

Comment:
This could be `https://en.wikipedia.org/` – the homepage doesn't have `/wiki/` for instance